components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/LogFilesResource.java

```diff
@@ -74,20 +79,44 @@ public Set<String> getMethods() {
     }
 
     @Override
-    public boolean invoke(MessageContext synCtx) {
+    public boolean invoke(MessageContext messageContext) {
```

Comment:
Why we changed the method name here? shall we keep the existing one in order to preserve the history?

Reply:
changed it for better readability. will undo it